includes/hub/admin/class-membership-plans.php

```diff
@@ -123,14 +132,44 @@ public static function get_membershp_plans_from_nodes() {
 						$network_pass_id = $meta->value;
 					}
 				}
+				if ( $network_pass_id && \Newspack_Network\Admin::use_experimental_auditing_features() ) {
```

Comment:
Nit: Might be worth namespacing this since we call `use_experimental_auditing_features` multiple times in this file

Reply:
Fixed in 121cc686236bac0458e5a8e811fa2380fca29171